azurerm/internal/services/healthcare/healthcare_service_data_source.go

```diff
@@ -37,9 +37,21 @@ func dataSourceHealthcareService() *schema.Resource {
 				Computed: true,
 			},
 
-			"cosmosdb_throughput": {
-				Type:     schema.TypeInt,
+			"cosmosdb_configuration": {
+				Type:     schema.TypeList,
 				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"throughput": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"key_vault_key_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
```

Comment:
Can we amend the data source to have properties matching the resource?

Reply:
Is this still relevant?

Reply:
We should maintain the existing `cosmosdb_throughput` attribute at least until v3.0, and I think it makes sense to have the same schema in the data source as in the corresponding resource.

Reply:
This should be fixed now. I might be wrong, but I think that the data source attributes should have their `Computed` properties set to `true`, and they don't necessarily need to be the same as their resource counterparts.

Reply:
Sorry that is correct of course, they should be Computed where they are not used as input properties. In terms of matching the names / nesting with the resource, it's not _mandatory_ but it's good to be consistent :)